pkg/operator/target_config_reconciler.go

```diff
@@ -213,6 +214,29 @@ func generateConfigMapString(requestedStrategies []deschedulerv1beta1.Strategy)
 			}
 		case "nodetaints", "removepodsviolatingnodetaints":
 			policy.Strategies["RemovePodsViolatingNodeTaints"] = deschedulerapi.DeschedulerStrategy{Enabled: true}
+		case "removepodshavingtoomanyrestarts":
+			podsHavingTooManyRestarts := deschedulerapi.PodsHavingTooManyRestarts{}
```

Comment:
Also, let's wait until https://github.com/kubernetes-sigs/descheduler/pull/276 gets merged.

Reply:
Agreed

Reply:
@ingvagabund the upstream PR has merged, so I updated this as well. Should be all good for a final pass